api/provision/csp-write-portfolio.ts

```diff
@@ -72,6 +72,8 @@ async function makeRequest(client: IAtatClient, request: ProvisionRequest): Prom
         clin: undefined,
       })),
     },
+    targetImpactLevel: request.targetImpactLevel,
```

Comment:
This won't be coming from the request - we'll need to pull it from Secrets Manager. Right now the schema is:

```
{
  "csp_name": {
    "uri": "https://atat-mock-server.azurewebsites.net"
  }
}
```

We'll probably want to modify that to:

```
{
  "csp_name": {
    "uri": "https://atat-mock-server.azurewebsites.net",
    "network": "IL2"
  }
}
```

And we may also want to document this structure in the HOTH API YAML. I can go ahead and update the records in SM if you're good with this.

Reply:
@jeffsegal-ccpo okay, yeah I forgot we were going to provide that in the secrets manager. That works.

Reply:
@jeffsegal-ccpo updated so SNOW doesn't know anything about `network` or `deadline` and updated HOTH.